de.tototec.cmvn/src/main/java/de/tototec/cmvn/MavenPomGenerator.java

```diff
@@ -4,11 +4,7 @@
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.io.PrintWriter;
-import java.util.HashMap;
-import java.util.LinkedHashMap;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
I'd prefer the non-wildcard imports as they make the class more readable when outside of an IDE.